examples/fault-injection/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-envoy
+    command: /usr/local/bin/envoy -c /etc/envoy.yaml
+    volumes:
+      - ./envoy.yaml:/etc/envoy.yaml
+      - ./runtime:/srv/runtime
+    networks:
+      - envoymesh
+    ports:
+      - 9211:9211
+      - 9901:9901
+  backend:
+    image: kennethreitz/httpbin@sha256:2c7abc4803080c22928265744410173b6fea3b898872c01c5fd0f0f9df4a59fb
```

Comment:
This sha hash is for current kennethreitz/httpbin:latest image. It seems the repository does not have tags to pin to specific version, so I'm using digest here.

Reply:
Isn't there an official HTTPbin? you could also add a simple python server FYI. There is also a small istio app that simply echos all the headers..

Reply:
The docker image seems HTTPbin's official one: https://github.com/requests/httpbin#officially-deployed-at

I could use the simple python server (d762488), but now I'm thinking using httpbin image is a much simpler.